bids_to_msd_dataset_conversion.py

```diff
@@ -0,0 +1,290 @@
+                                                        utils.add_suffix(filename, args.label_suffix))
+
+                        else:
+                            # if a session or contrast is not included, skip it
+                            continue
+
+                        # each session will be stored as image_01, image_01, etc. and label_01, label_02, etc.
+                        temp_data[f"image_0{session_ctr+1}"] = image_file
+                        temp_data[f"label_0{session_ctr+1}"] = label_file
+
+                        # increment the session counter
+                        session_ctr += 1
+
+                else:
+                    # print("Found that --group_by_sessions is NOT set. Using all sessions to create independent image and label files ...") 
+                    temp_data = {}
+
+                    # check if only the specific sessions have to be included
+                    if args.include_sessions is not None and sessionID in args.include_sessions and contrast_suffixID in args.include_contrasts:
+                        image_file = os.path.join(root, subjectID, sessionID, datatype, filename)
+                        label_file = os.path.join(root, 'derivatives', 'labels', subjectID, sessionID, datatype, 
+                                                    utils.add_suffix(filename, args.label_suffix))
+
+                    elif args.include_sessions is None and contrast_suffixID in args.include_contrasts:
+                        # assuming all sessions have to be included
+                        image_file = os.path.join(root, subjectID, sessionID, datatype, filename)
+                        label_file = os.path.join(root, 'derivatives', 'labels', subjectID, sessionID, datatype,
+                                                    utils.add_suffix(filename, args.label_suffix))
+                    else:
+                        # if a session or contrast is not included, skip it
+                        continue
+
+                    # store in a temp dictionary
+                    temp_data[f"image"] = image_file
+                    # NOTE: Currently only works for single contrast (because label for that will be available)
+                    temp_data[f"label"] = label_file
+                    temp_list.append(temp_data)
+
+        # only append the temp_list if grouping by sessions is set
+        temp_list.append(temp_data) if args.group_by_sessions or args.group_by_contrasts else None
+        # print(temp_list)
+
+    params[name] = temp_list
+
+
+# get the number of training, validation and test samples depend on the depending on the grouping/sessions/contrasts chosen
+params["numTraining"] = len(params["training"])
+params["numValidation"] = len(params["validation"])
+params["numTest"] = len(params["test"])
+
+final_json = json.dumps(params, indent=4, sort_keys=True)
+jsonFile = open(os.path.join(args.path_out, f"dataset.json"), "w")
```

Comment:
If `args.path_out` does not exist, the script fails. There are two possible workarounds:
1. exit the script if the `args.path_out` does not exist
2. check if `args.path_out` exists, if not, create it